compiler/vmdef.nim

```diff
@@ -75,6 +75,7 @@ type
     opcSubStr, opcParseFloat, opcConv, opcCast,
     opcQuit,
     opcNarrowS, opcNarrowU,
+    opcSpreadSignBit, # if 32 bit convert 0x0000000080000000 -> 0xffffffff80000000 to make intVal negative
```

Comment:
This operation is called sign extension so what about giving the OP a name like `opcSignExtend`?